src/lib/util/options.mli

```diff
@@ -1051,60 +1051,93 @@ val get_is_gui : unit -> bool
-val get_fmt_std : unit -> Format.formatter
-(** Default to [Format.std_formatter] *)
+      Default to [Format.std_formatter]. *)
 
-(** Value specifying the formatter used to output errors *)
-val get_fmt_err : unit -> Format.formatter
-(** Default to [Format.err_formatter] *)
+  val get_fmt_err : unit -> Format.formatter
+  (** Value specifying the formatter used to output errors.
 
-(** Value specifying the formatter used to output warnings *)
-val get_fmt_wrn : unit -> Format.formatter
-(** Default to [Format.err_formatter] *)
+      Default to [Format.err_formatter]. *)
 
-(** Value specifying the formatter used to output debug informations *)
-val get_fmt_dbg : unit -> Format.formatter
-(** Default to [Format.err_formatter] *)
+  val get_fmt_wrn : unit -> Format.formatter
+  (** Value specifying the formatter used to output warnings.
 
-(** Value specifying the formatter used to output model *)
-val get_fmt_mdl : unit -> Format.formatter
-(** Default to [Format.std_formatter] *)
+      Default to [Format.err_formatter]. *)
 
-(** Value specifying the formatter used to output unsat core *)
-val get_fmt_usc : unit -> Format.formatter
-(** Default to [Format.std_formatter] *)
+  val get_fmt_dbg : unit -> Format.formatter
+  (** Value specifying the formatter used to output debug informations.
 
-(** Set [fmt_std] accessible with {!val:get_fmt_std} *)
-val set_fmt_std : Format.formatter -> unit
+      Default to [Format.err_formatter]. *)
 
-(** Set [fmt_err] accessible with {!val:get_fmt_err} *)
-val set_fmt_err : Format.formatter -> unit
+  val get_fmt_mdl : unit -> Format.formatter
+  (** Value specifying the formatter used to output models.
 
-(** Set [fmt_wrn] accessible with {!val:get_fmt_wrn} *)
-val set_fmt_wrn : Format.formatter -> unit
+      Default to [Format.std_formatter]. *)
 
-(** Set [fmt_dbg] accessible with {!val:get_fmt_dbg} *)
-val set_fmt_dbg : Format.formatter -> unit
+  val get_fmt_usc : unit -> Format.formatter
+  (** Value specifying the formatter used to output unsat cores.
 
-(** Set [fmt_mdl] accessible with {!val:get_fmt_mdl} *)
-val set_fmt_mdl : Format.formatter -> unit
+      Default to [Format.std_formatter]. *)
 
-(** Set [fmt_usc] accessible with {!val:get_fmt_usc} *)
-val set_fmt_usc : Format.formatter -> unit
+  val set_std : t -> unit
+  (** Set [fmt_std] accessible with {!val:get_fmt_std} *)
+
+  val set_err : t -> unit
+  (** Set [fmt_err] accessible with {!val:get_fmt_err} *)
+
+  val set_wrn : t -> unit
+  (** Set [fmt_wrn] accessible with {!val:get_fmt_wrn} *)
+
+  val set_dbg : t -> unit
+  (** Set [fmt_dbg] accessible with {!val:get_fmt_dbg} *)
+
+  val set_mdl : t -> unit
+  (** Set [fmt_mdl] accessible with {!val:get_fmt_mdl} *)
+
+  val set_usc : t -> unit
+  (** Set [fmt_usc] accessible with {!val:get_fmt_usc} *)
+end
+
+(** Print message as comment in the corresponding output format *)
+val print_output_format: Format.formatter -> string -> unit
```

Comment:
I know this function is not new in the PR, but if "Print a message as comment in the corresponding output format" is an accurate description of what is doesn, maybe we could take the opportunity to rename it into `pp_comment`?